prog/ai/inference_endpoint_nexus.rb

```diff
@@ -55,9 +55,9 @@ def self.assemble(project_id:, location:, boot_image:, name:, vm_size:, storage_
       ubid = InferenceEndpoint.generate_ubid
       internal_project = Project[Config.inference_endpoint_service_project_id]
       fail "No project configured for inference endpoints" unless internal_project
-      firewall = internal_project.firewalls_dataset.where(location: location).where(Sequel[:firewall][:name] => "inference-endpoint-firewall").first
-      fail "No firewall named 'inference-endpoint-firewall' configured for inference endpoints in #{location}" unless firewall
-      subnet_s = Prog::Vnet::SubnetNexus.assemble(internal_project.id, name: ubid.to_s, location: location, firewall_id: firewall.id)
+      firewall = internal_project.firewalls_dataset.where(location_id: location_id).where(Sequel[:firewall][:name] => "inference-endpoint-firewall").first
```

Comment:
```suggestion
      firewall = internal_project.firewalls_dataset.where(location_id:).where(Sequel[:firewall][:name] => "inference-endpoint-firewall").first
```

Could omit hash values here and elsewhere.